cmd/config/configcobra/cmds.go

```diff
@@ -1,17 +1,17 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-// Package cmds provides a target for embedding the config command group in another
+// Package configcobra provides a target for embedding the config command group in another
 // cobra command.
-package cmds
+package configcobra
 
 import (
 	"strings"
 
 	"github.com/spf13/cobra"
-	"sigs.k8s.io/kustomize/cmd/config/cmd"
-	"sigs.k8s.io/kustomize/cmd/config/cmddocs/api"
-	"sigs.k8s.io/kustomize/cmd/config/cmddocs/tutorials"
+	"sigs.k8s.io/kustomize/cmd/config/internal/cmd"
```

Comment:
while you're here and in the mood -

how about renaming `internal/cmd` to `internal/commands`, as is done here 
https://github.com/kubernetes-sigs/kustomize/tree/master/kustomize

This reserves `cmd` for top level un-importable mains, per https://github.com/golang-standards/project-layout#cmd

everything else looks good.  could pull this first if you like

Reply:
I'll update it now.